examples/low-level-api/list-offers.py

```diff
@@ -4,13 +4,18 @@
 from datetime import datetime, timezone
 import json
 import sys
+import pathlib
+
 
 from yapapi import props as yp
 from yapapi.log import enable_default_logger
 from yapapi.props.builder import DemandBuilder
 from yapapi.rest import Configuration, Market, Activity, Payment  # noqa
 
-from examples import utils
+examples_dir = pathlib.Path(__file__).resolve().parent.parent
+sys.path.append(str(examples_dir))
+import utils
+
```

Comment:
@krunch3r76 Three blank lines is too much for `black` (our code linter), please remove one of them.